src/plots/cartesian/axes.js

```diff
@@ -249,7 +249,12 @@ axes.handleTickDefaults = function(containerIn, containerOut, coerce, axType, op
 
             if(!tickFormat && axType !== 'date') {
                 coerce('showexponent', showAttrDflt);
-                coerce('exponentformat');
+                coerce('exponentbase');
+
+                var expBase = containerOut.exponentbase,
```

Comment:
Try

```js
var expBase = coerce('exponentbase');
```

which coerces 'exponentbase' and returns `containerOut.exponentbase`.